tests/test_parquet.py

```diff
@@ -163,7 +163,33 @@ def test_write_roundtrip(tmpdir):
     tmpdir = str(tmpdir)
     arr = dak.from_awkward(ak.from_iter(data), 2)
     to_parquet(arr, tmpdir)
-    arr = dak.from_parquet(tmpdir)
+    dak.from_parquet(tmpdir)
+
+
+def test_nested_columns(tmpdir):
```

Comment:
Sorry, this doesn't belong here, but in the parquet segfault discussion #140 . It can be removed if problematic.

Reply:
Doesn't bother me to have have this in- seems like a good test to have anyway!